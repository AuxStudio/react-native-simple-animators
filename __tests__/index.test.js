import React from 'react';
import renderer from 'react-test-renderer';
import { Animated } from 'react-native';

import Animator from '../';

describe('Animator', () => {
  it('imports the module', () => {
    expect(
      renderer.create(
        <Animator
          type="height"
          animatedValue={new Animated.Value(0)}
          interpolation={{ inputRange: [0, 1], outputRange: [0, 100], extrapolate: 'clamp' }}
        />,
      ),
    ).toMatchSnapshot();
  });
});
