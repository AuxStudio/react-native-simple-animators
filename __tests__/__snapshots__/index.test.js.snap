// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Animator imports the module 1`] = `
<View
  style={
    Object {
      "height": 0,
    }
  }
/>
`;
